trace/tests/unit/v1/test__gapic_v1.py

```diff
@@ -20,34 +20,30 @@
 from google.api_core import grpc_helpers
 
 
-class _Base(object):
+class Test__TraceAPI(unittest.TestCase):
     project = "PROJECT"
 
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.trace.v1._gapic import _TraceAPI
+
+        return _TraceAPI
+
     def _make_one(self, gapic_client=None, handwritten_client=None):
         from google.cloud.trace_v1.gapic import trace_service_client
 
-        channel = grpc_helpers.ChannelStub()
         if gapic_client is None:
-            gapic_client = trace_service_client.TraceServiceClient(channel=channel)
+            gapic_client = mock.create_autospec(trace_service_client.TraceServiceClient)
```

Comment:
I assume you're using a mock instead of channel stub to simplify the tests? Seems reasonable to me, if so.

Reply:
Yes, as well as the fact that passing in a channel is deprecated.